### DISCUSSION TITLE
Feature: Provide item explore list beside folders for easier access when there are too many files

### DISCUSSION THREAD

**Discussion by ryzenway** [toeverything/AFFiNE#8801] [Ideas]
### Description

When there are too many files in folder it becomes difficult to browse and see all files. Also shifting to other folder becomes difficult. Feature to show just folders and when folder selected then show all files within list beside folder would be helpful. Here is how a plugin in obsidian is doing. It is very important to work with large files.
Obsidian File Tree Alternative https://www.youtube.com/watch?v=fbz8IZtXuUE

### Use case

- to explore large number of files within folder
- switch to other folders fast
- not clutter browser

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by hwangdev97** [comment]
Agreed, we'll provide a more convenient folder tree in the team workspace.
<img width="922" alt="image" src="https://github.com/user-attachments/assets/e562c0d3-71d3-4fd2-94fc-b4cbe313c27a">

**Reply by ryzenway** [reply]
Is there any date when it will be implemented
